cpg_utils/hail_batch.py

```diff
@@ -12,29 +12,30 @@
 from cloudpathlib import CloudPath
 from cloudpathlib.anypath import to_anypath
 
+from config import get_config
+
 
 GCLOUD_AUTH_COMMAND = (
     'gcloud -q auth activate-service-account --key-file=/gsa-key/key.json'
 )
 
+_config = get_config()
```

Comment:
This means that users should make sure to set `CPG_CONFIG_PATH` before they import cpg_utils. Which makes it 
difficult to allow external users to configure their runs with command line parameters alone, without environment variables. Maybe it's okay, as long as we document this clearly in the readme. They'll anyway have to configure somehow paths to references, images, inputs, Hail, etc. But ideally it should be possible both through configs and command line, e.g. with [click-config](https://github.com/EverythingMe/click-config). I don't have good ideas though.

Reply:
Good point, thanks! I've added a `set_config_path` method and don't call `get_config()` at the `hail_batch` module scope anymore. Do you think that's better?

Please let me know if there's a way to make pylint happy without disabling the warnings for the globals that are used for lazy initialization.

Reply:
I don't think there is a way to make it happy, global variables are just generally discouraged. But an alternative would be explicit singleton classes, which is also too wordy and not very pythonic.